nameable.rb

```diff
@@ -0,0 +1,5 @@
+class Nameable
+  def correct_name
+    raise NotImplementedError
+  end
+end
```

Comment:
```suggestion
class Nameable
    def correct_name
        raise NotImplementedError, "#{self.class} has not implemented method '#{__method__}'"
    end
end

```

- [ ] Your method 'correct_name' should raise an error, It should look like the above 👍🏿